plugins/mac-capture/window-utils.m

```diff
@@ -48,7 +55,6 @@ bool find_window(cocoa_window_t cw, obs_data_t *settings, bool force)
 
 	if (!cw->window_name.length && !cw->owner_name.length)
 		goto invalid_name;
-
```

Comment:
Please remove this line change from the diff.

Reply:
Whoops, don't know how that got there. Removed it and re-ran clang after not realizing I was forgetting the `-i` flag... everything should be good now